proto-build/src/main.rs

```diff
@@ -236,7 +235,6 @@ fn compile_luna_classic_protos_and_services(out_dir: &Path) {
     tonic_build::configure()
         .build_client(true)
         .build_server(false)
-        .format(true)
```

Comment:
any reasons to remove format?

Reply:
It doesn't exist in 0.7 tonic-build: https://docs.rs/tonic-build/0.7.2/tonic_build/struct.Builder.html#implementations
See: https://github.com/hyperium/tonic/commit/d6c0fc112b2288a080fd0a727453b24d666e3d79